docs/src/components/TableOfContents.tsx

```diff
@@ -1,11 +1,56 @@
 import React from "react";
 import Scrollspy from "react-scrollspy";
 import styled, { css } from "styled-components";
+import { v4 as uuidv4 } from "uuid";
 
-export interface TocItemObject {
+import { slugify } from "../utils/common";
+
+const DEPTH = 3;
```

Comment:
This excludes `h4` and `h5` from the table of contents, i. e. it only goes 3 levels deep.